docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -214,6 +214,14 @@ This means that the number of rows will drive the height of the grid and consequ
 
 {{"demo": "pages/components/data-grid/rendering/AutoHeightGrid.js", "bg": "inline"}}
 
+### Infinite loading
+
+The grid provides a `onRowsScrollEnd` prop that can be used to load additional rows when the scroll reaches the bottom of the viewport area.
+
+In addition the area in which the callback provided to the `onRowsScrollEnd` is called can be change using `scrollEndThresholdHeight`
+
+{{"demo": "pages/components/data-grid/rendering/InfiniteLoadingGrid.js", "bg": "inline"}}
```

Comment:
Maybe I'm missing something, but the demo is not working. DataGrid stops virtualization after 100 rows. Either we should add pagination or use XGrid.

Reply:
I also noticed that. Basically to use the full potential of that feature one would need to use the `XGrid`. I can change it to `XGrid` but that seems counterintuitive as the feature is free but we showcase it on a paid component.

Reply:
No strong preferences. I simply think that stopping at 100 rows isn't enough (current). Either we have the pagination to quick in, or we use XGrid. @dtassone what do you think we should do?

Reply:
I'm more in favour of using `XGrid` then because pagination isn't really an infinite loading scenario in my mind.

Reply:
Yes this is an XGrid feature only

Reply:
Ok, I wasn't aware that the infinite loading is an XGrid feature. I'll make the additional checks required for it.

Reply:
> Yes this is an XGrid feature only

If it's the case, then we need to prune the props from the DataGrid exposed API and documentation.

Reply:
Updated, the feature is now Pro